src/quo/components/list_items/dapp/view.cljs

```diff
@@ -1,22 +1,22 @@
 (ns quo.components.list-items.dapp.view
   (:require
-    [quo.components.icon :as icons]
     [quo.components.list-items.dapp.style :as style]
     [quo.components.markdown.text :as text]
-    [quo.foundations.colors :as colors]
     [quo.theme :as quo.theme]
     [react-native.core :as rn]
     [react-native.fast-image :as fast-image]))
 
 (defn view
-  [{:keys [dapp action on-press on-press-icon] :as props}]
+  [{:keys [dapp on-press right-component] :as props}]
   (let [theme                  (quo.theme/use-theme)
         [pressed? set-pressed] (rn/use-state false)
         on-press-in            (rn/use-callback #(set-pressed true))
         on-press-out           (rn/use-callback #(set-pressed false))]
     [rn/pressable
      {:style        (style/container (assoc props :pressed? pressed?))
-      :on-press     on-press
+      :testID       "dapp-pressable"
```

Comment:
we should avoid testID's unless really needed, accessibility labels much better.
https://testing-library.com/docs/queries/about/